sim/battle.ts

```diff
@@ -1665,6 +1678,12 @@ export class Battle extends Dex.ModdedDex {
 		this.started = true;
 		this.p2.foe = this.p1;
 		this.p1.foe = this.p2;
+		if (this.gameType === 'multi') {
+			this.sides[0].ally = this.sides[2];
+			this.sides[1].ally = this.sides[3];
+			this.sides[2].ally = this.sides[0];
+			this.sides[3].ally = this.sides[1];
```

Comment:
This design supposes exactly one ally. I'd rather not have an `.ally` entry at all.

Reply:
Are you saying we ought to support 6 player triple battles?

Reply:
Maybe one day!

Reply:
No reason to write the code to preclude that, at least.